### PR TITLE
Add Signus: The Artefact Wars

### PR DESCRIPTION
**Pull request by nextghost**
Signus: The Artefact Wars is a Czech turn-based strategy wargame originally released in 1998. The original developers have open-sourced the game on SourceForge back in 2002 and I've finished porting it to Linux and modern Windows earlier this year.

The game has no official website and the only Wikipedia page about it is in Czech so the "official" URL points to a fan website with some English content.

### CODE REVIEW

File: originals/s.yaml [hunk 1]
```diff
@@ -335,6 +335,17 @@
...
 
+- name: 'Signus: The Artefact Wars'
+  external:
+    website: 'http://signus.paan.cz/'
+  platform:
+  - Windows
+  meta:
+    genre:
+    - Turn-Based Strategy
+    themes:
```
Comment (cxong) [inline originals/s.yaml:346]:
```suggestion
    theme:
```

File: games/s.yaml [hunk 2]
```diff
@@ -434,6 +434,21 @@
...
   url: http://soaos.sourceforge.net/
 
+- name: 'Signus: The Artefact Wars'
+  type: remake
+  originals:
+  - 'Signus: The Artefact Wars'
+  repo: 'https://github.com/signus-game/signus'
+  development: complete
+  status: playable
+  lang: C++
```
Comment (cxong) [inline games/s.yaml:444]:
```suggestion
  lang: C++
  framework:
  - SDL2
```

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/s.yaml [hunk 3]
```diff
@@ -434,6 +434,23 @@
...
```
Comment (cxong) [inline games/s.yaml:444]:
```suggestion
  lang:
  - C++
```

File: games/s.yaml [hunk 4]
```diff
@@ -434,6 +434,23 @@
...
+  framework:
+  - SDL2
+  license: GPL2
```
Comment (cxong) [inline games/s.yaml:447]:
```suggestion
  license:
  - GPL2
```

**Review by cxong — CHANGES_REQUESTED** [review]

**Comment by nextghost** [comment]
I've fixed the `lang` and `license` structures, squashed the whole PR into a single commit and rebased it on current master.

**Review by cxong — APPROVED** [review]

**Pull request merged**